src/Microsoft.ML.StandardTrainers/Standard/ModelStatistics.cs

```diff
@@ -204,7 +204,7 @@ private protected virtual VersionInfo GetVersionInfo()
 
     // REVIEW: Reconcile with the stats in OLS learner.
     /// <summary>
-    /// The statistics for linear predictor.
```

Comment:
>predictor [](start = 34, length = 9)

Thanks @soorajshajahan@ymail.com !
Could you also make the same change in line 72?